scripts/mcptam_install.bash

```diff
@@ -81,35 +97,34 @@ install_dependencies()
 	install_toon
 	install_libcvd
 	install_gvars3
-	rm -rf build_deps
 	cd ..
+	rm -rf build_deps
```

Comment:
This is why I mkdir -p, it should fail graciously, but I just realized this may not be the case for the subfolders. Will fix.